test/integration/user_login_test.rb

```diff
@@ -40,9 +40,9 @@ def setup
     assert root_path, current_path
   end
 
-  test 'a logged in user cannot go to admin items path' do
+  test 'a logged in user cannot go to admin projects path' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.